pkg/node/refresh.go

```diff
@@ -0,0 +1,19 @@
+package node
+
+import (
+	"time"
+
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+const (
+	networkStateRefresh = time.Minute
+)
+
+// NodeNetworkStateRefresh add some jitter to to the refresh rate so it does
+// not hit apiserver at the same time, the jittering is implemented using
```

Comment:
It is clear how it's implemented from the body

Reply:
But you don't see the body at godocs.

Reply:
Do we apply this rule to other bigger functions?

I won't fight for this, but linking documentation of the implementation feels like a burden on maintenance.

Reply:
I will copy paste what Jitter does.